x-pack/plugins/reporting/server/lib/tasks/execute_report.ts

```diff
@@ -502,7 +502,9 @@ export class ExecuteReportTask implements ReportingTask {
 
             stream.end();
 
+            this.logger.debug(`Begin waiting for the stream's pending callbacks...`);
             await finishedWithNoPendingCallbacks(stream);
+            this.logger.info(`The stream's pending callbacks have completed.`);
```

Comment:
should we add jobId or something like that? Doesn't look like it is part log meta

Reply:
I agree. We're using `this.logger` which is initialized at plugin setup time, so the logging tags don't include the context of the job ID - this is an oversight. The logger should always have the context of the job ID when it is available. Pushed 1d1f223cedf